### DISCUSSION TITLE
How to update the transmission web GUI and CLI?

### DISCUSSION THREAD

**Discussion by skaveesh** [transmission/transmission#4775] [Q&A]
I have updated from transmission 2 to 4 today but my CLI & web GUI is still stuck at version 2.94.

```
pi@raspberrypi:/opt/transmission-setup/transmission-4.0.0/build $ transmission-cli -V
transmission-cli 2.94 (d8e60ee44f)
pi@raspberrypi:/opt/transmission-setup/transmission-4.0.0/build $ transmission-daemon -V
transmission-daemon 4.0.0 (280ace12f8)
pi@raspberrypi:/opt/transmission-setup/transmission-4.0.0/build $ transmission-remote -V
transmission-remote 4.0.0 (280ace12f8)
pi@raspberrypi:/opt/transmission-setup/transmission-4.0.0/build $ transmission-create -V
transmission-create 4.0.0 (280ace12f8)
pi@raspberrypi:/opt/transmission-setup/transmission-4.0.0/build $ transmission-edit -V
transmission-edit 4.0.0 (280ace12f8)
pi@raspberrypi:/opt/transmission-setup/transmission-4.0.0/build $ transmission-show -V
transmission-show 4.0.0 (280ace12f8)
```

how do I update these? any help would be appreciated.

I have attached the cmake logs here:
[log-transmission4-installation.txt](https://file.io/XNaQn0t8T5Xc)

linux version info:
```
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
```

**Comment by mikedld** [comment]
You seem to have done the upgrade manually, w/o it being provided by your system package manager. Since you were building from source, please note that transmission-cli is deprecated for a long time now and is not built by default. If you still want to build it, you can pass `-DENABLE_CLI=ON` to cmake when configuring.

**Reply by skaveesh** [reply]
Thank you @mikedld  for the reply. I wasn't aware that cli was deprecated. 
Could you help me on updating the GUI from 2.94 to 4?

[Image-web GUI still at 2.94](https://i.imgur.com/p2WuJcI.png)

**Reply by pfg21rus** [reply]
please add this option to README.MD section ### Building a Transmission

**Reply by ile6695** [reply]
@pfg21rus readme already says that cli is deprecated. In docs/ there are more detailed build instructions that do contain `-DENABLE_CLI`
